frontend/src/css/PopupSelectImage.styl

```diff
@@ -36,3 +36,6 @@
     &__validate
       margin-inline-start auto
       margin-inline-end 0
+
+[dir=rtl] .PopupSelectImage
```

Comment:
I'd suggest to put this above the ``.PopupSelectImage`` class and create a ``[dir=ltr]`` section too
This way we can keep track of changes made on ``ltr`` and ``rtl`` easily